Helper/ConfigHelper.php

```diff
@@ -334,11 +336,20 @@ public function makeSeoRequest($storeId = null)
      * @param $storeId
      * @return bool
      */
-    public function isLoggingEnabled($storeId = null)
+    public function isLoggingEnabled($storeId = null): bool
     {
         return $this->configInterface->isSetFlag(self::LOGGING_ENABLED, ScopeInterface::SCOPE_STORE, $storeId);
     }
 
+    /**
+     * @param $storeId
+     * @return bool
+     */
+    public function isProfilingEnabled($storeId = null): bool
```

Comment:
You know I fought myself about this also. I wanted to use `profiler` everywhere but was also trying to stay consistent with the legacy method convention of `isLoggingEnabled` (when it probably should be `isLoggerEnabled`) but didn't want to change that since it was a public method. But no point repeating the sins of the past! I'll update it so the profiling part is at least consistent.